yaksh/templates/yaksh/course_modules.html

```diff
@@ -15,22 +15,17 @@
 {% block main %}
 
 <div class="container">
-<div class="row justify-content-md-center yakshwell ">
+<div class="row justify-content-md-center yakshwell">
     <div class="col-md-10 bg-light card">
-        <div class="row align-items-center">
+        <div class="row align-items-center" style="height: 100px;">
             <div class="col h4">  {{ course.name }}  </div>
-            <div class="col-md-3 ml-auto yakshwell">
-                <div class="row align-items-center"> 
-                    <div class="col-sm-6"> Overall
-                    Course</br>Completion :
-                    </div>
-                    <div class="col-sm-5">
-                    <span class="progress">
-                      <span class="progress-bar bg-warning " role="progressbar" style="width: {{ course_percentage }}%; color: black;" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100">{{ course_percentage }} % completed</span>
-                    </span>
-                    </div>
-                </div>
-            </div>   
+            <div class = "col">
+                {% if course.has_lessons %}&nbsp;&nbsp;&nbsp;
+                <a href="{{URL_ROOT}}/exam/download_course/{{course.id}}" data-toggle="tooltip" title="Download course content" class="btn btn-primary pull-right">
```

Comment:
Can we start using ```url``` template tag instead of given path?

Reply:
@prathamesh920 For some reason django url tag is causing problem. It throws NoReverseMatch error.

Reply:
Can you share your code and error with me

Reply:
@prathamesh920 
```
<a href="{% url 'download_course' course.id %}" data-toggle="tooltip" title="Download course content" class="btn btn-primary pull-right">Download Course</a>
```

Reply:
Use namespace yaksh.
So in tag call ```'yaksh:download _coursr'```
Above should work.

Reply:
Usually namespaces are used in apps urls.py as global variable. I didn't see that in root urls.py